---
string name # name of the closest landmark
